### DISCUSSION TITLE
[v4] Want to add more data in the json response

### DISCUSSION THREAD

**Discussion by Neewd** [adonisjs/core#1867] [Help] [answered]
Hey,

I think the serializer would solve my  problem but I found the documentation quite vague, so I'll explain what my need. 

I have a manyThrough relation ship in a model, like: Picture taken in a ---> Location who belongs to a --> Country. 
I can now have the number of pictures, but I would like to add this number when I retrieve all the Country like, Ok this country have 19 pictures taken through the locations. 

How can I do this ? 
Thanks :D

**Comment by sanconley** [comment, marked as answer]
You can set up the manyThrough in reverse.

```javascript

// Country.js

 pictures () {
    return this.manyThrough('App/Models/Picture', 'location')
  }

// Controller somewhere
 const pictures = await Country.query().with('pictures').fetch()
```
